src/iter/mod.rs

```diff
@@ -110,6 +124,13 @@ impl<'data, I: 'data + ?Sized> IntoParallelRefIterator<'data> for I
     }
 }
 
+
+/// `IntoParallelRefMutIterator` implements the conversion to a [`ParallelIterator`] providing mutable references to the data.
+///
+/// This is a parallel version of the `iter_mut()` method 
+/// defined by various collections.
```

Comment:
We should probably also mention that this trait is automatically implemented `for I where &mut I: IntoParallelIterator`.  In other words, one should probably go implement that rather than implementing this trait directly.